mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
>Could be possible to use Blender bundled Python to run unit tests?

Sorry, i don't know how to do that. It would be great if someone can do that !

>But it could be good to implement an operator to find existing meshes and attach them to mmd_root.

Yeah, that will make more sense i think. :smile: But it might not be necessary since we can do that by using blender's tool. Maybe join meshes from different mmd model is more useful ! And you can create another PR if you want to do that. :)